### ISSUE TITLE
Should blue graphics shift position left and right between Composite (Idealized) and RGB?

### ISSUE THREAD

**Issue by Sunspark-007** [AppleWin/AppleWin#1298]
A good example to see this in action would be the starting main menu of the Oregon Trail. It has two blue banners/decorations on it.

Shifting between those two literally shifts the blue pixels left and right while the white text stays in place and does not shift.

**Comment by tomcw** [comment]
>Should blue graphics shift position left and right between Composite (Idealized) and RGB?

Yes, because blue pixels are with the HGR high bit set, so there's a "half-dot shift" for "Composite (Idealized)" mode.
In RGB mode, there's no "half-dot shift".

In "Composite (Idealized)" mode, you can see this by doing the following:
- HGR
- CALL-151
- 2000:C0 0

This last line plots a blue pixel at offset 6 (ie. with high bit=1), but the 2nd byte has it's high bit=0, so there's no shift. This truncates the pixel to be half an HGR pixel wide. In RGB mode, the pixel isn't truncated.

To see the full (non-truncated) blue pixel, then do this:
- 2001:80

If you are in RGB mode, then doing either 2001:0 or 2001:80 has no affect on the blue pixel's width.

**Comment by tomcw** [comment]
Closing as question answered (ie. expected behaviour).
